projects/demo/src/modules/experimental/card/examples/6/index.html

```diff
@@ -0,0 +1,51 @@
+<div tuiCard>
+    <h2
+        tuiTitle
+        [style.max-width]="'100%'"
```

Comment:
Let's bundle this into tuiTitle and tuiSubtitle inside tuiCard. I don't think it ever makes sense for them to overflow.

Reply:
fixed